client/src/main.tsx

```diff
@@ -4,13 +4,16 @@ import { BrowserRouter } from 'react-router-dom';
 import App from './App.tsx';
 import './index.css';
 import AuthProvider from "./context/AuthContext";
-
+import { GoogleOAuthProvider } from '@react-oauth/google';
 
 ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
   // <React.StrictMode>
   <BrowserRouter>
     <AuthProvider>
-      <App />
+      <GoogleOAuthProvider clientId="120111886097-er99p69quna208pvuupimkjm073pb996.apps.googleusercontent.com">
+        <App />
+      </GoogleOAuthProvider>
```

Comment:
it is best practice to store clientId or any sensitive information on the server-side or in a .env file